Open Judge System/LocalWorker/OJS.LocalWorker/OjsSubmissionProcessingStrategy.cs

```diff
@@ -277,6 +277,7 @@ private void SetSubmissionToProcessed()
             try
             {
                 this.submission.Processed = true;
+                this.submission.CompletedExecutionOn = DateTime.Now;
```

Comment:
Is StartedExecutionOn set to Now or UtcNow?

Reply:
It is DateTime.Now